presto-orc/src/main/java/com/facebook/presto/orc/writer/SliceDictionaryBuilder.java

```diff
@@ -132,7 +132,7 @@ private long getHashPositionOfElement(Block block, int position)
                 // Doesn't have this element
                 return hashPosition;
             }
-            else if (segmentedSliceBuilder.equals(slicePosition, block, position, length)) {
+            if (segmentedSliceBuilder.equals(slicePosition, block, position, length)) {
```

Comment:
Applying the same change as suggested in, https://github.com/prestodb/presto/pull/16905#discussion_r738979083